pkgs/top-level/all-packages.nix

```diff
@@ -24376,8 +24376,6 @@ with pkgs;
 
   quattrocento-sans = callPackage ../data/fonts/quattrocento-sans {};
 
-  r3rs = callPackage ../data/documentation/rnrs/r3rs.nix { };
```

Comment:
Doesn't match the removed file.

Reply:
oops, fixed. The initial set of these changes was performed in an automated way.
I'd maybe like to put the script into nixpkgs eventually, so this could be automated periodically (e.g. before each release).